packages/ember-glimmer/lib/ember-metal-views/index.js

```diff
@@ -13,6 +15,17 @@ class DynamicScope {
   }
 }
 
+function setupRerenderLoop(renderer, view) {
+  // let LAST_TAG_VALUE = CURRENT_TAG.value();
+  run.backburner.on('begin', () => {
+    run.backburner.schedule('render', () => {
```

Comment:
we don't need to recreate this anon function every time
it also needs to go away when the root view is gone, maybe the renderer should track root views.